content/blog/2019-11-17-salut-bucharest.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Salut Bucharest
+author: sathya
+type: post
+date: 2019-11-17T04:51:13+00:00
+url: /2019/11/17/salut-bucharest/
+categories:
+  - Life
+tags:
+  - bucharest
+  - romania
+  - career
+  - immigration
+
+---
+
+
+It's been a while since I posted anything, so lets get this started again. As I had tweeted couple of weeks back, I have moved to Bucharest, Romania. 
+
+<!--more-->
+
+
+{{< tweet 1182251512334077952 >}}
+
+{{< tweet 1184008163546284032 >}}
+
+So some of the obvious questions would be:
+
+- Why Romainia?
+- How did it happen?
+
+Let's take on them one by one:
+
+### Why Romania?
+
+To which I say why not? I've been looking at moving out for a while now, and the US was never an option, especially with the current visa situation(and Jo not being able to work with a dependent visa). Europe is what Jo and me had been looking at since a while. I've been working with my colleagues from Bucharest, Romania since past couple of couple of years and this gave me a chance to bring up a internal transfer request within my team. With the approvals in place, we were finally able to start the process. However, as with immigration & inter-country transfers, this was just the start of a long road of lots of paperwork and documentation shuttle.
+
+### How did it happen?
+
+Immigration and work visas are never easy, and Romania's visa/work permit process wasn't a piece of cake either. Before the visa process could begin, I had to get a work permit - which can be only done by the employer. And for applying the work permits, I had to get a "Criminal Record" aka PCC/Police Clearance Certificate, Transcripts from my University, my Degree certificate and my previous experience certificates - and I had to get them [Apostilled](https://en.wikipedia.org/wiki/Apostille_Convention) too!
+
+One of the many TIL moments was about Apostille.. 
+
+> A certification under the terms of the Apostille Convention is called an apostille (from Latin post illa and then French: a marginal note) or Hague Apostille. It is an international certification comparable to a notarisation in domestic law, and normally supplements a local notarisation of the documen
+
+The good thing was most of these were well calatalogued and saved(I have almost everything digitised with multiple backups) and the ones that needed to be done, namely the transcripts and PCC, I could apply for these online. 
```

Comment:
`catalogued`? There's a bit here where there could be a space before the bracket again...

Reply:
lol at "calatalogued". Fixed